ext/civicrm_admin_ui/managed/SavedSearch_Administer_MembershipStatusRules.mgd.php

```diff
@@ -0,0 +1,236 @@
+<?php
+use CRM_CivicrmAdminUi_ExtensionUtil as E;
```

Comment:
@aydun is there a reason for these 2 managed entity files they seem to be doing the same thing but are named differently and have different cleanup policies always v unused